client/post-editor/editor-publish-date/style.scss

```diff
@@ -51,21 +51,17 @@
 	}
 
 	// down-arrow
-	&::after {
-		@include noticon( '\f431', 22px );
-
-		display: block;
-		width: 20px;
-		line-height: #{ $header-height - 1 }px;
-		color: rgba($gray, 0.5);
-
+	.gridicon.editor-publish-date__header-chevron {
+		fill: $gray-darken-20;
 		position: absolute;
-		right: 14px;
-		top: 0px;
+		top: 10px;
+		right: 13px;
+		vertical-align: text-bottom;
+		transition: transform 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
 	}
 
 	// up-arrow
-	.editor-publish-date.is-open &::after {
+	.editor-publish-date.is-open & .gridicon.editor-publish-date__header-chevron {
```

Comment:
Not sure about the use of the `&` here... it's a bit confusing to get. Maybe write it like:

```sass
 	.gridicon.editor-publish-date__header-chevron {
		fill: $gray-darken-20;
		position: absolute;
		top: 10px;
		right: 13px;
		vertical-align: text-bottom;
		transition: transform 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
		
		.editor-publish-date.is-open & {
		  transform: rotate(180deg);
	  }
	}
```

I'm not sure if the way you wrote it is actually "wrong" though. Just something I haven't seen.

Reply:
Yea, you're probably right. I was being lazy and just tried to make it work with the existing CSS. I'll refactor a bit.